exp/tests/test_response_views.py

```diff
@@ -986,7 +985,7 @@ def test_get_appropriate_children_in_child_csv_as_previewer(self):
         content = csv_response.content.decode("utf-8")
         csv_reader = csv.reader(io.StringIO(content), quoting=csv.QUOTE_ALL)
         csv_body = list(csv_reader)
-        csv_headers = csv_body.pop(0)
```

Comment:
likewise here there's a side effect of pop() even though we don't use the assigned csv_headers

Reply:
Yup, I got a bit delete happy there, my apologies.  It should be back now.